tst/schedulers/test_searchers.py

```diff
@@ -0,0 +1,135 @@
+# Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License").
+# You may not use this file except in compliance with the License.
+# A copy of the License is located at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# or in the "license" file accompanying this file. This file is distributed
+# on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+# express or implied. See the License for the specific language governing
+# permissions and limitations under the License.
+import pytest
+from datetime import datetime
+import numpy as np
+
+from syne_tune.optimizer.baselines import (
+    RandomSearch,
+    GridSearch,
+    BayesianOptimization,
+    ASHA,
+    HyperTune,
+    SyncHyperband,
+    DEHB,
+    BOHB,
+    SyncBOHB,
+    BORE,
+    KDE,
+)
+from syne_tune.config_space import (
+    choice,
+    ordinal,
+    config_space_size,
+)
+from syne_tune.backend.trial_status import Trial
+from syne_tune.optimizer.scheduler import SchedulerDecision
+
+
+# Does not contain ASHABORE, because >10 secs on CI
+# TODO: Dig more, why is ASHABORE more expensive than BORE here?
+@pytest.mark.timeout(10)
+@pytest.mark.parametrize(
+    "scheduler_cls, multifid",
+    [
+        (RandomSearch, False),
+        (GridSearch, False),
+        (BayesianOptimization, False),
+        (ASHA, True),
+        (HyperTune, True),
+        (SyncHyperband, True),
+        (DEHB, True),
+        (BOHB, True),
+        (SyncBOHB, True),
+        (BORE, False),
+        (KDE, False),
+    ],
+)
+def test_allow_duplicates_or_not(scheduler_cls, multifid):
```

Comment:
seems natural to move `allow_duplicates` here

Reply:
What do you mean?

Reply:
add another `parametrize` line with `allow_duplicates` before the function